src/main/java/com/base64/d/StudyGuideChecker.java

```diff
@@ -8,127 +8,120 @@
 
 import java.io.IOException;
 import java.util.Base64;
-import java.util.Date;
 import java.util.HashSet;
 import java.util.Set;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 public class StudyGuideChecker {
-    int count = 0;
-    public int totalLinksFound = 0;
-    String url3 = "https://content.infinitecampus.com/sis/latest/study-guide/ad-hoc-filters-letters-and-data-viewer";
-    String decodedText = "";
-    String iframeSrc = "";
-    String date = new Date().toString();
 
-    String titleOfStudyGuide = "";
-    Set<String> studyGuideLinks = new HashSet<>();
-    Set<String> badLinks = new HashSet<>();
+    public void checkStudyGuides() {
+        int count = 0;
+        int totalLinksFound = 0;
 
-    WriteToFile writeToFile = new WriteToFile();
+        String decodedText = "";
+        String iframeSrc = "";
+        String titleOfStudyGuide = null;
 
-    public void checkStudyGuides() {
+        Set<String> studyGuideLinks = new HashSet<>();
+        Set<String> badLinks = new HashSet<>();
 
-        for (String link2 : AllStudyGuideLinks.ALL_STUDY_GUIDES) {
+        WriteToFile writeToFile = new WriteToFile();
+
+        for (String studyGuideLink : AllStudyGuideLinks.ALL_STUDY_GUIDES) {
+            // Find all links within a Study Guide
             try {
-//
-//          Connect to the study guide url
```

Comment:
It's good to have code comments, especially any area where the functionality is not obvious. The double forward slashes for these comments appear to be created with the "comment out code" shortcut in IntelliJ, which starts the slashes at the first character of the line. This is more useful when commenting out a block of code (to really help draw attention to the full block that is commented out, since all of the characters are on the left side of the screen, which is usually blank). What is preferred for code comments is to have the slashes justified with the rest of the code. You can see below how the two forward slashes have been moved to be more inline with the code.